packages/dds/merge-tree/src/endOfTreeSegment.ts

```diff
@@ -177,11 +185,11 @@ export class EndOfTreeSegment extends BaseEndpointSegment implements ISegment, I
 		};
 	}
 
-	get ordinal() {
+	get ordinal(): string {
 		// just compute an arbitrarily big ordinal
 		// we base it on the depth of the tree
 		// to ensure it is bigger than all ordinals in
 		// the tree, as each layer appends to the previous
-		return String.fromCharCode(0xffff).repeat(this.endpointSegmentProps().depth);
+		return String.fromCodePoint(0xffff).repeat(this.endpointSegmentProps().depth);
```

Comment:
though it's not a huge deal and I think the behavior here is correct, I would have leaned toward disabling the lint rule when dealing with ordinals. The intent of this lint rule is probably "you probably don't support unicode correctly if you're using this and likely have bugs", but for ordinals we obviously aren't actually ever dealing with user input, we're just using it for dictionary order.

i think it's fine as is, but worth a comment. whatever you go with, just be consistent that all logic related to ordinals uses a consistent method.